src/stories/epics/createForm.stories.tsx

```diff
@@ -8,7 +8,7 @@ import { withRedux } from '../decorators';
 import { sampleProject1, sampleUser1 } from '../fixtures';
 
 export default {
-  title: 'Epics/CreateEpicModal/Component',
+  title: 'Epics/CreateEpicModal/Example',
```

Comment:
```suggestion
  title: 'Epics/CreateForm/Example',
```

Reply:
I'm fine with this change. Depends whether we want the navigation to exactly mirror component names, or to be more readable.